rebac-admin-backend/v1/error.go

```diff
@@ -53,6 +53,8 @@ func NewValidationError(message string) error {
 
 // ErrorResponseMapper is the basic interface to allow for error -> http response mapping
 type ErrorResponseMapper interface {
+	// MapError maps an error into a Response. If the method is unable to map the
+	// error (e.g., the error is unknown), it can just return nil.
```

Comment:
suggestion: "it **must** return nil"
I think this is need because we rely on the return value to be nil in case "we" have to take care of the mapping